src/library/Uno.Material/Controls/Divider.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace Uno.Material.Controls
+{
+	public partial class Divider : Control
+	{
+		public string SubHeader
+		{
+			get { return (string)GetValue(SubHeaderProperty); }
+			set { SetValue(SubHeaderProperty, value); }
+		}
+
+		public static readonly DependencyProperty SubHeaderProperty =
+			DependencyProperty.Register("SubHeader", typeof(string), typeof(Divider), new PropertyMetadata(string.Empty));
+
+		public Brush SubHeaderForeground
```

Comment:
Should just reuse the foreground of the control. If we have to customize that, expose SubHeaderStyle for that.

Reply:
@XiaotianNetlift He is already using the foreground for the divider in the template.

Reply:
@Xiaoy312 Yeah like @agneszitte-nventive said, I'm already using the foreground property for the rectangle. If I expose a style for the SubHeader, I won't still be able to change the textblock foreground, since all of our style are black by default.